AnsiConsole/src/mnita/ansiconsole/participants/AnsiConsoleStyleListener.java

```diff
@@ -60,42 +60,30 @@ public void lineGetStyle(LineStyleEvent event) {
 			}
 		}
 
+		// no position means nothing to do
+		if (positions.length == 0)
```

Comment:
That's not true.
Even if there are no escapes in the current line, we still need to return attributes set in a previous line.
Think
```
Hello \033[91m red
this should still be red, even if there are no escapes
and also red \033[m normal
```